### PR TITLE
Update Metadata + Screenshots of Activities

### PR DESCRIPTION
**Pull request by Gabbar8055**

### CODE REVIEW

File: activity/activity.info [hunk 1]
```diff
@@ -6,3 +6,7 @@ icon = activity-blockparty
 activity_version = 8
 license = MIT
 show_launcher = yes
+summary = https://github.com/sugarlabs/block-party-activity
```
Comment (vipulgupta2048) [inline activity/activity.info]: @Gabbar8055 Thanks for the patch.
The summary as described [here](https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/bundle/__init__.py) should give a basic description of the activity. Please fill the appropriate metadata.

Comment (Gabbar8055) [reply activity/activity.info]: Done!

**Comment by vipulgupta2048** [comment]
Thanks. Good progress. 
@Gabbar8055 Can you tell me how you reproduced the images? I don't understand the small size and the Sugar toolbar missing from the top. 
@quozl Can you review further?

**Comment by Gabbar8055** [comment]
Thanks @vipulgupta2048 
I am using Vmware to run sugar on my system and captured screenshots from snipping tool.
The Sugar toolbar do not apper when you run the block party activity thats why its missing from top.

**Comment by quozl** [comment]
@Gabbar8055, thanks, reviewed.  Please fix blurring.

@vipulgupta2048, thanks for mentioning me, I was not watching this repository.

**Comment by quozl** [comment]
@Gabbar8055, thanks, the blur is reduced.

I've compared against running the activity on Ubuntu 16.04 with Sugar 0.112.

* [ ] there's no "Press ESC to exit" message on your screenshots, and I've no idea why; are you using the latest source code from the repository here or a bundle instead?  The message was added in 6af29a1ccf59962b342a638503f63ed4fc052e3b.
* [ ] there's no initial screen; the one that has "SELECT LEVEL: $(n) enter to start",
* [ ] there's a light blue horizontal bar at the top of your screenshots, but it doesn't happen when I run the activity,
* [ ] the blur is still there around the edges of the blocks; yours;

your

mine;

mine

I understand you've finished GCI.  I'm happy to take over from here, if you're busy with other things, just let me know.

**Comment by pro-panda** [comment]
Looks like @Gabbar8055 is busy

**Comment by quozl** [comment]
Thanks for testing.  None of what you say is relevant to this pull request, which was make-work to update metadata and screenshots as part of yet another attempt to replace ASLO.  But it is relevant to issue https://github.com/sugarlabs/block-party-activity/issues/4, so I'll link that issue here just to add to the confusion.

Yes, BlockParty-7 does work with the much older OLPC OS builds which were based on earlier Sugar, but the reason it worked was that CSound provided a TCP/IP network server and the activity connected to the server to make sound.  Later version of CSound in Fedora, which was adopted by later OLPC OS builds, removed that capability, probably for some unspecified security reason.

In order to remain operational, BlockParty is obliged to use CSound by direct API instead of network.  That's what issue https://github.com/sugarlabs/block-party-activity/issues/4 is to track.

**Pull request closed without merging**